README.md

```diff
@@ -112,490 +76,16 @@ Secures your IaC files. Critical to keeping your cloud deployment safe and secur
 
 The extension also applies [JFrog File Spec JSON schema](https://raw.githubusercontent.com/jfrog/jfrog-cli/master/schema/filespec-schema.json) on the following file patterns: `**/filespecs/*.json`, `*filespec*.json` and `*.filespec`. Read more about JFrog File specs [here](https://www.jfrog.com/confluence/display/JFROG/FileSpec).
```

Comment:
This part is for more experienced users, so I'd rather move it to GitBook.

Reply:
I removed this line altogether @Or-Geva.